include/seqan/basic/debug_test_system.h

```diff
@@ -326,13 +322,15 @@ inline const char * toCString(Demangler<T> const & me)
 
 // Set default for SEQAN_ENABLE_DEBUG.
 #ifndef SEQAN_ENABLE_DEBUG
-  #ifdef NDEBUG
-    #define SEQAN_ENABLE_DEBUG 0
-  #else  // #ifdef NDEBUG
-    #define SEQAN_ENABLE_DEBUG 1
-  #endif  // #ifdef NDEBUG
+#define SEQAN_ENABLE_DEBUG 0
 #endif  // #ifndef SEQAN_ENABLE_DEBUG
 
+#if !SEQAN_ENABLE_DEBUG
+#define NDEBUG 1
+#else
+#undefine NDEBUG
```

Comment:
should be `#undef` I think

please double-check the influence of turning the flags on and off.